weather/src/components/PrintWeather.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export function PrintWeather (city) {
```

Comment:
```suggestion
export function Weather (props) {
```
Имя компонентов, как и название файлов должно быть существительным и предметным
Все что приходит в компонент является props